packages/fluentui/react-northstar/src/components/Menu/menuContext.ts

```diff
@@ -2,11 +2,13 @@ import { createContext } from '@fluentui/react-context-selector';
 import { ComponentVariablesInput } from '@fluentui/styles';
 import * as React from 'react';
 import { Accessibility } from '@fluentui/accessibility';
+import { MenuItemProps } from './MenuItem';
 
 export type MenuContextValue = {
   activeIndex: number;
   variables: ComponentVariablesInput;
-  onItemSelect: (e: React.KeyboardEvent | React.MouseEvent, itemIndex: number) => void;
+  onItemClick: (e: React.KeyboardEvent | React.MouseEvent, itemProps: MenuItemProps) => void;
+  onItemSelect: (e: React.KeyboardEvent | React.MouseEvent, itemProps: MenuItemProps) => void;
```

Comment:
```suggestion
  onItemSelect: (e: React.KeyboardEvent | React.MouseEvent, itemIndex: number) => void;
```

It seems that we should be able to keep the previous signature